examples/site/testing_storage/storage_unit_test.cc

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_storage_unit_test]
+#include <google/cloud/functions/cloud_event.h>
+#include <boost/log/core.hpp>
+#include <boost/log/sinks/sync_frontend.hpp>
+#include <boost/log/sinks/text_ostream_backend.hpp>
+#include <boost/shared_ptr.hpp>
+#include <gmock/gmock.h>
+#include <nlohmann/json.hpp>
+#include <memory>
+#include <sstream>
+
+namespace gcf = ::google::cloud::functions;
+extern void hello_world_storage(gcf::CloudEvent event);
```

Comment:
Any reason why we don't use a `.h` for this declaration?

Reply:
I wanted to keep the "Hello World" examples minimal, they do not need a `.h`, they are just a `.cc` that gets compiled into a container.  Introducing a `.h` complicates the hello world example, and does not make this code any easier to grok for a customer: they need to read the `.h` and the `*_test.cc` to understand the test.